CMakeLists.txt

```diff
@@ -796,7 +796,7 @@ JOIN("-I${DEPENDENCIES_INCLUDE_DIRS}" " -I" PKGCONFIG_OSRM_INCLUDE_FLAGS)
 JOIN("${ENGINE_LIBRARIES}" " " PKGCONFIG_OSRM_DEPENDENT_LIBRARIES)
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/pkgconfig.in libosrm.pc @ONLY)
-install(FILES ${PROJECT_BINARY_DIR}/libosrm.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+install(FILES ${PROJECT_BINARY_DIR}/libosrm.pc DESTINATION ${PKGCONFIG_LIBRARY_DIR}/pkgconfig)
```

Comment:
Is the extra `pkgconfig` required?

Looking at

https://github.com/Project-OSRM/osrm-backend/blob/0b6eb85106425a1c1c0f08fe6a8e1936245b85e1/cmake/pkgconfig.in#L3

I don't think so - could you check to make sure we're in sync here.

Reply:
@daniel-j-h `.pc` files go into `/usr/local/lib/pkgconfig`, the `PKGCONFIG_LIBRARY_DIR` is where we put `libosrm.a`